tests/integration/kubernetes/gha-run.sh

```diff
@@ -452,15 +452,31 @@ function deploy_nydus_snapshotter() {
 		  misc/snapshotter/base/nydus-snapshotter.yaml \
 		  'data.FS_DRIVER' \
 		  "proxy" --style=double
+		# Disable to read snapshotter config from configmap
+		yq write -i \
+		  misc/snapshotter/base/nydus-snapshotter.yaml \
+		  'data.ENABLE_CONFIG_FROM_VOLUME' \
+		  "false" --style=double
+	elif [ "${PULL_TYPE}" == "host-share-image-block" ]; then
```

Comment:
This block cannot be tested on this PR. So I suggest to move these changes to https://github.com/kata-containers/kata-containers/pull/7837

Here we leave only a placeholder like:

```
elif [ "${PULL_TYPE}" == "host-share-image-block" ]; then
 echo "Skip deploying nydus for the pull type `host-share-image-block` until the PR https://github.com/kata-containers/kata-containers/pull/7837 is prepared"
 return
else
..
..
```

Reply:
Done.